docs/nvidia-guide.md

```diff
@@ -0,0 +1,78 @@
+# NVIDIA graphics cards
+_Driver API and source code available in [`liquidctl.driver.nvidia`](../liquidctl/driver/nvidia.py)._
+
+Support for these cards in only available on Linux.
+
+Additional requirements must also be met:
+
+- optional Python dependency `smbus` is available
+- `i2c-dev` kernel module has been loaded
+- specific unsafe features have been opted in
+- r/w permissions to card-specific `/dev/i2c-*` devices
```

Comment:
I also forgot to add a new rule for the EVGA card in [`extra/linux/71-liquidctl.rules`](https://github.com/jonasmalacofilho/liquidctl/blob/master/extra/linux/71-liquidctl.rules).

Reply:
That device is also included PR #228 (I know it should have probably been here but, not a big deal)